notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T231-analysis.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>Is there a particular reason of why we are using CSV files? All the data should be in the EFD and we should query the EFD instead of using CSV files.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
The CSV files were used in the previous analysis. I agree to upgrade to an EFD-based version. Let's keep this code as `LVV-T231-analysis-offline` and add a new file as "LVV-T231-analysis-offline".